packages/flutter_tools/test/integration/flutter_run.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This needs to end in `_test.dart` in order to get run (it's be ignored right now)

Reply:
Good spot! Fixed here -> https://github.com/flutter/flutter/pull/21438